### PR TITLE
chore(web-components): update readme and add quick start docs

### PR DESCRIPTION
**Pull request by chrisdholt**
## Description
Updates the README for the Web Components package for accuracy and adds initial developer quick start pages.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: change/@fluentui-web-components-8fcdb3e8-703f-4a42-a618-ba66f348b53d.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-web-components-8fcdb3e8-703f-4a42-a618-ba66f348b53d.json:1]: ## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31416&commitId=800a38da4ca94f1d482a6fada482badc62c5fabe&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Size - Dark Mode.normal.chromium.png|2| Changed|
|Accordion.Size.normal.chromium.png|2| Changed|
|Accordion.Size - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Avatar</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Color - RTL.normal.chromium.png|1| Changed|
|Avatar.Active Appearance - Dark Mode.normal.chromium.png|2| Changed|
|Avatar.Color.normal.chromium.png|1| Changed|
|Avatar.Active Appearance.normal.chromium.png|10| Changed|
|Avatar.Active.normal.chromium.png|5| Changed|

</details>


  <details><summary><b>Badge</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge.Default.normal.chromium.png|1| Changed|
|Badge.Appearance - Dark Mode.normal.chromium.png|15| Changed|
|Badge.Appearance.normal.chromium.png|1| Changed|
|Badge.Shape.normal.chromium.png|1| Changed|
|Badge.Size.normal.chromium.png|6| Changed|
|Badge.Color.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Button</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Large Icon Only - Dark Mode.hover.chromium.png|3| Changed|
|Button.Large Icon Only - Dark Mode.pressed.chromium.png|5| Changed|
|Button.Large Icon Only.default.chromium.png|5| Changed|
|Button.Large Icon Only - Dark Mode.default.chromium.png|4| Changed|
|Button.Large With Icon - Dark Mode.default.chromium.png|4| Changed|
|Button.Large Icon Only.pressed.chromium.png|3| Changed|
|Button.Large Icon Only.hover.chromium.png|4| Changed|
|Button.Large With Icon - RTL.pressed.chromium.png|3| Changed|
|Button.Large With Icon - Dark Mode.hover.chromium.png|3| Changed|
|Button.Large With Icon.hover.chromium.png|4| Changed|
|Button.Large With Icon.pressed.chromium.png|3| Changed|
|Button.Large With Icon - RTL.hover.chromium.png|4| Changed|
|Button.Large With Icon.default.chromium.png|5| Changed|
|Button.Large With Icon - RTL.default.chromium.png|5| Changed|
|Button.Large With Icon - Dark Mode.pressed.chromium.png|5| Changed|

</details>


  <details><summary><b>Divider</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.Vertical With Svg.normal.chromium.png|4| Changed|
|Divider.With Svg - RTL.normal.chromium.png|3| Changed|
|Divider.With Svg.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>MenuList</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuList.Checkbox - RTL.2nd selected.chromium.png|1| Changed|
|MenuList.Checkbox.2nd selected.chromium.png|1| Changed|
|MenuList.Checkbox With Icons.1st selected.chromium.png|4| Changed|
|MenuList.Checkbox With Icons - RTL.1st selected.chromium.png|4| Changed|
|MenuList.Checkbox With Icons.normal.chromium.png|4| Changed|
|MenuList.Checkbox With Icons - RTL.normal.chromium.png|4| Changed|
|MenuList.Checkbox With Icons - RTL.2nd selected.chromium.png|5| Changed|
|MenuList.Radio With Icons - RTL.normal.chromium.png|4| Changed|
|MenuList.Checkbox With Icons.2nd selected.chromium.png|5| Changed|
|MenuList.Radio With Icons - RTL.1st selected.chromium.png|4| Changed|
|MenuList.Radio With Icons.normal.chromium.png|4| Changed|
|MenuList.With Icons.normal.chromium.png|4| Changed|
|MenuList.Radio With Icons - RTL.2nd selected.chromium.png|4| Changed|
|MenuList.Radio With Icons.1st selected.chromium.png|4| Changed|
|MenuList.Radio With Icons.2nd selected.chromium.png|4| Changed|
|MenuList.With Icons.hover menuitem.chromium.png|3| Changed|

</details>


  <details><summary><b>Slider</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Size Small.rightArrow.chromium.png|1| Changed|
|Slider.Size Small.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked.hover.chromium.png|1| Changed|
|Switch.Checked - RTL.hover.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>


  <details><summary><b>TextInput</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextInput.Size Large.normal.chromium.png|2| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Review by eljefe223 — APPROVED** [review]

File: packages/web-components/README.md [hunk 2]
```diff
@@ -1,31 +1,33 @@
 # Fluent UI Web Components
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![npm version](https://badge.fury.io/js/%40fluentui%2Fweb-components.svg)](https://badge.fury.io/js/%40fluentui%2Fweb-components)
+[![npm version](https://img.shields.io/npm/v/@fluentui/web-components/beta?style=flat-square)](https://www.npmjs.com/package/@fluentui/web-components/v/beta)
```
Comment (janechu) [inline packages/web-components/README.md:4]: Duplicate of https://github.com/microsoft/fluentui/pull/31371?

File: packages/web-components/README.md [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
+yarn add @fluentui/web-components@beta
 ```
 
-Within your JavaScript or TypeScript code, you can then import library APIs like this:
+**NPM**
 
-```ts
-import { FluentAnchor } from '@fluentui/web-components';
+```sh
+npm i @fluentui/web-components@beta
```
Comment (janechu) [inline packages/web-components/README.md:23]:
```suggestion
npm i --save @fluentui/web-components@beta
```

File: packages/web-components/src/_docs/developer/quick-start.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+**Yarn**
+
+```sh
+yarn add @fluentui/web-components
+```
+
+**NPM**
+
+```sh
+npm i @fluentui/web-components
```
Comment (janechu) [inline packages/web-components/src/_docs/developer/quick-start.stories.mdx:18]:
```suggestion
npm i --save @fluentui/web-components
```

File: packages/web-components/src/_docs/developer/theming.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,61 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Developer/Theming" />
+
+# Theming
+
+The Fluent UI Theme is represented by a set of tokens. Each token resolves to a single value which can be assigned to a CSS property.
+
+```tsx
```
Comment (janechu) [inline packages/web-components/src/_docs/developer/theming.stories.mdx:9]:
```suggestion
```ts
```

**Review by janechu — APPROVED** [review]

File: packages/web-components/README.md (hunk 3 above)
Comment (chrisdholt) [reply packages/web-components/README.md:23]: Do we need this in modern versions of NPM (greater than 5.0.0)?

File: packages/web-components/src/_docs/developer/quick-start.stories.mdx (hunk 4 above)
Comment (chrisdholt) [reply packages/web-components/src/_docs/developer/quick-start.stories.mdx:18]: Do we need this in modern versions of NPM (greater than 5.0.0)?

Comment (janechu) [reply packages/web-components/src/_docs/developer/quick-start.stories.mdx:18]: Good point, I'm out of date!

File: packages/web-components/README.md (hunk 2 above)
Comment (chrisdholt) [reply packages/web-components/README.md:4]: yeah, i figure while i'm here...

**Pull request merged**